danspeech/DanSpeechRecognizer.py

```diff
@@ -87,10 +107,80 @@ def update_decoder(self, lm=None, alpha=None, beta=None, labels=None, beam_width
             else:
                 self.decoder = GreedyDecoder(labels=self.labels, blank_index=self.labels.index('_'))
 
+    def enable_streaming(self):
+        self.streaming = True
+        self.greedy_decoder = GreedyDecoder(labels=self.labels, blank_index=self.labels.index('_'))
+        self.audio_parser = InferenceSpectrogramAudioParser(audio_config=self.audio_config)
+
+    def disable_streaming(self):
+        self.streaming = False
+        self.audio_parser = SpectrogramAudioParser(self.audio_config)
+        self.iterating_transcript = ""
+        self.full_output = []
+        self.spectrograms = []
+
+    def streaming_transcribe(self, recording, is_last, is_first):
+        recording = self.audio_parser.parse_audio(recording, is_last)
+
+        transcript = ""
+        if len(recording) != 0:
+            # ToDO: Remove but keep here for now
```

Comment:
extend explanation, what is affected by the toDo, the entire assertion?

Reply:
Will also be cleaned! :)